### PR TITLE
I updated from webpack to vite and Laravel from 8 to 10

### PR DESCRIPTION
**Pull request by leandroximenes**
I've updated the project, transitioning from Webpack to Vite. Additionally, I've upgraded the Laravel version from 8 to 10, ensuring compatibility with the latest features and improvements. These changes aim to optimize the project's development environment with the current version.

I made all these modifications in the devContainer because it's my preferred environment, and I have already committed the changes. I temporarily removed files like ESLint and Prettier just to simplify the process for now, but we can reintroduce them later after configuring in the devContainer.

My next steps involve upgrading the tests and implementing debugging with devContainer.

I'm thrilled to contribute to the project.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,25 +5,22 @@
...
-        "fakerphp/faker": "^1.9.1",
-        "fruitcake/laravel-cors": "^2.0",
-        "guzzlehttp/guzzle": "^7.0.1",
-        "inertiajs/inertia-laravel": "^0.6.4",
-        "laravel/framework": "^8.65",
-        "laravel/sanctum": "^2.11",
-        "laravel/tinker": "^2.5",
-        "league/glide-laravel": "^1.0"
+        "php": "^8.1",
+        "fruitcake/laravel-cors": "dev-develop",
```
Comment (ordago) [inline composer.json:9]: fruitcake/laravel-cors is not in Laravel 10. It was removed in Laravel 9.

Here is the PR that made the change:

https://github.com/laravel/laravel/pull/5825

File: package.json [hunk 2]
```diff
@@ -1,41 +1,25 @@
...
-        "ssr:serve": "node public/js/ssr.js"
+        "dev": "vite",
+        "build": "vite build"
     },
     "dependencies": {
-        "@inertiajs/inertia": "^0.11.0",
-        "@inertiajs/inertia-vue3": "^0.6.0",
-        "@inertiajs/progress": "^0.2.7",
-        "@inertiajs/server": "^0.1.0",
+        "@inertiajs/progress": "^0.1.2",
```
Comment (ordago) [inline package.json:9]: The latest version of inertia doens't need `@inertiajs/progress` to be installed:

https://inertiajs.com/upgrade-guide#progress

File: public/build/assets/Create-1f9edace.js
Comment (ordago) [inline public/build/assets/Create-1f9edace.js:1]: It seems the built asset were accidentally committed.

**Comment by leandroximenes** [comment]
Thanks for your feedback! I'll address the mentioned libraries and remove built assets from commits. I temporarily removed Prettier and ESLint to avoid confusion but will reintegrate them in the next commit. Your insights are valuable, and I appreciate your review.

**Comment by leandroximenes** [comment]
Hello @ordago, 

I would like to inform you that I have completed the required corrections, and the revised version is now ready for your review

**Comment by ordago** [comment]
> I would like to inform you that I have completed the required corrections, and the revised version is now ready for your review

Hey @leandroximenes 

I'm sorry if I gave the impression that I'm a maintainer of the repo, I'm not. I just follow the repo and provided feedback. I wish you luck with the PR.

**Comment by leandroximenes** [comment]
Thank you, @ordago 

I appreciate your feedback, and I'm learning from it. I believe that this way, I can contribute a consistent and relevant (PR). Although you are not the maintainer, I appreciate your time spent with me.

**Comment by jessarcher** [comment]
Thanks! Ended up letting Shift handle this in #209.

**Pull request closed without merging**